README.md

```diff
@@ -38,17 +38,15 @@ Please report security issues to our [HackerOne](https://hackerone.com/homebrew/
 
-Homebrew/homebrew-core's lead maintainer is [ilovezfs](https://github.com/ilovezfs).
-
-Homebrew/brew's other current maintainers are [Dominyk Tiller](https://github.com/DomT4), [Claudia](https://github.com/claui), [Michka Popoff](https://github.com/imichka), [Shaun Jackman](https://github.com/sjackman), [Chongyu Zhu](https://github.com/lembacon), [commitay](https://github.com/commitay), [Vitor Galvao](https://github.com/vitorgalvao), [ilovezfs](https://github.com/ilovezfs), [JCount](https://github.com/jcount), [Misty De Meo](https://github.com/mistydemeo), [Gautham Goli](https://github.com/GauthamGoli), [Markus Reiter](https://github.com/reitermarkus) and [William Woodruff](https://github.com/woodruffw).
+Homebrew/brew's other current maintainers are [Dominyk Tiller](https://github.com/DomT4), [Claudia](https://github.com/claui), [Michka Popoff](https://github.com/imichka), [Shaun Jackman](https://github.com/sjackman), [Chongyu Zhu](https://github.com/lembacon), [commitay](https://github.com/commitay), [Vitor Galvao](https://github.com/vitorgalvao), [JCount](https://github.com/jcount), [Misty De Meo](https://github.com/mistydemeo), [Gautham Goli](https://github.com/GauthamGoli), [Markus Reiter](https://github.com/reitermarkus) and [William Woodruff](https://github.com/woodruffw).
 
 Homebrew/brew's Linux support (and Linuxbrew) maintainers are [Michka Popoff](https://github.com/imichka) and [Shaun Jackman](https://github.com/sjackman).
 
-Homebrew/homebrew-core's other current maintainers are [Dominyk Tiller](https://github.com/DomT4), [Claudia](https://github.com/claui), [Michka Popoff](https://github.com/imichka), [Shaun Jackman](https://github.com/sjackman), [Chongyu Zhu](https://github.com/lembacon), [commitay](https://github.com/commitay), [Izaak Beekman](https://github.com/zbeekman), [Sean Molenaar](https://github.com/SMillerDev), [Viktor Szakats](https://github.com/vszakats), [FX Coudert](https://github.com/fxcoudert), [JCount](https://github.com/jcount), [Misty De Meo](https://github.com/mistydemeo) and [Tom Schoonjans](https://github.com/tschoonj).
+Homebrew/homebrew-core's other current maintainers are [Dominyk Tiller](https://github.com/DomT4), [Claudia](https://github.com/claui), [Michka Popoff](https://github.com/imichka), [Shaun Jackman](https://github.com/sjackman), [Chongyu Zhu](https://github.com/lembacon), [commitay](https://github.com/commitay), [Izaak Beekman](https://github.com/zbeekman), [Sean Molenaar](https://github.com/SMillerDev), [Jan Viljanen](https://github.com/javian), [Viktor Szakats](https://github.com/vszakats), [FX Coudert](https://github.com/fxcoudert), [JCount](https://github.com/jcount), [Misty De Meo](https://github.com/mistydemeo) and [Tom Schoonjans](https://github.com/tschoonj).
```

Comment:
`other current maintainers` doesn't make as much sense without the prior reference to a lead maintainer. I'd possibly just drop the `other` so it reads `Homebrew/homebrew-core's current maintainers are`?

Reply:
@DomT4 Good point, pushing.

Reply:
@DomT4 Actually: this wording is used for `Homebrew/brew` too which has no lead of it's own. Additionally, I'm not mentioned in either sets of lists. Thoughts?

Reply:
That's true. It looks like those two were separated by the changes in https://github.com/Homebrew/brew/commit/ee5a073b057724e2cca238050db98d00c78e6c1c. I'd be temped to add you to both lists _(presuming you want to be in both lists)_ and drop the `other` across the board, personally, but it's a minor enough thing if you wanted to leave it as-is I doubt many people would notice, heh.

Reply:
Cool, will leave it as-is for now  then. Thanks anyway @DomT4 ❤️